tests/target/issue-3343/two.rs

```diff
@@ -10,3 +10,5 @@ use total_len_is::_51___;
 
 /// this is a comment to test is_sugared_doc property
 use core::convert;
+
+#[foooooo] #[barrrrr] use total_len_is::_50______;
```

Comment:
Hmm, I feel that putting multiple attributes on a single line is too much. I would like to enable this configuration option only when there is a single attribute.